### PR TITLE
API-4765 oauth-proxy moved to lighthouse-oauth-proxy repo

### PR DESCRIPTION
**Pull request by crolarlibertyva**
Oauth Proxy moved to https://github.com/department-of-veterans-affairs/lighthouse-oauth-proxy

### CODE REVIEW

**Comment by beaugrantham-va** [comment]
Do the cicd scripts need to be updated to remove oauth-proxy references?

https://github.com/department-of-veterans-affairs/vets-saml-proxy/blob/API-4765-strip-out-oauth-proxy/saml-proxy/cicd/buildspec-release.yml#L28

**Comment by crolarlibertyva** [comment]
> Do the cicd scripts need to be updated to remove oauth-proxy references?
> 
> https://github.com/department-of-veterans-affairs/vets-saml-proxy/blob/API-4765-strip-out-oauth-proxy/saml-proxy/cicd/buildspec-release.yml#L28

Eventually. Not until https://github.com/department-of-veterans-affairs/devops/pull/9197 and a subsequent one that I'll get in for saml-proxy gets merged and applied.

**Comment by beaugrantham-va** [comment]
What about the readme? Now or later?

I'm good with either, just making sure to track it.

**Comment by crolarlibertyva** [comment]
> What about the readme? Now or later?
> 
> I'm good with either, just making sure to track it.

I can get that in now, Just note that until we get more  tf changes applied, changes not in saml-;proxy will not trigger ci.

File: README.md [hunk 1]
```diff
@@ -1,5 +1,3 @@
-# SAML/OAuth Proxies
-
 ## SAML Proxy
```
Comment (beaugrantham-va) [inline README.md:1]: Minor comment, but can you adjust the header hierarchy?

**Review by beaugrantham-va — APPROVED** [review]

**Pull request merged**